Ports/openrct2/package.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='openrct2'
+version='0.4.3'
+auth_type='sha256'
+depends=(
+    'SDL2'
+    'curl'
+    'flac'
+    'fontconfig'
+    'freetype'
+    'libicu'
+    'libogg'
+    'libpng'
+    'libvorbis'
+    'libzip'
+    'openssl'
+    'speexdsp'
+    'zlib')
+files="https://github.com/OpenRCT2/OpenRCT2/archive/refs/tags/v${version}.tar.gz ${port}-${version}.tar.gz 1269759b3a90717f379cd375f30629e3f24d3fdeb0c54cc1dca03676e64731ef"
+useconfigure='true'
+workdir="OpenRCT2-${version}"
+configopts=(
+	"-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
+	"-DCMAKE_INSTALL_PREFIX=${DESTDIR}/usr/local/"
+    "-DCMAKE_BUILD_TYPE=Release"
+	"-DDISABLE_DISCORD_RPC=ON"
+	"-DDISABLE_GOOGLE_BENCHMARK=ON"
+    "-DWITH_TESTS=OFF"
+    "-DDISABLE_OPENGL=ON")
```

Comment:
Same here.

Reply:
There also appears to be a mixup of spaces and tabs in this place specifically. We usually indent using 4 spaces only.